rsconf/component/devbox.py

```diff
@@ -57,11 +57,11 @@ def internal_build_compile(self):
     def internal_build_write(self):
         from rsconf import systemd
 
-        if "user_name" not in self:
+        if self.name == "devbox":
```

Comment:
make a conditional func so clear what's going on. Could be called is_main_instance

Reply:
7e5d358